app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  after_create :update_counter
+  after_delete :update_counter
+
+  def update_counter
+    post = Post.find(post_id)
+    post.update(likes_counter: post.likes.count)
+  end
+end
```

Comment:
- [ ] you can streamline your likes counter too! Check out this example:

```
# Your model code
class Like < ApplicationRecord
  belongs_to :post
  belongs_to :user

  after_create :increment_post_likes_count
  after_destroy :decrement_post_likes_count

  # Rest of your model code

  private

  def increment_post_likes_count
    post.increment!(:likes_counter)
  end

  def decrement_post_likes_count
    post.decrement!(:likes_counter)
  end
end
```

#### Don't forget to use the correct callback ` after_destroy`  instead of `after_delete` and now you make make your method private too.